src/domains/exchange/components/ExchangeCard/ExchangeCard.tsx

```diff
@@ -1,59 +1,59 @@
 import { Card } from "app/components/Card";
-import { Icon } from "app/components/Icon";
-import { Image } from "app/components/Image";
+import { DropdownOption } from "app/components/Dropdown";
 import React from "react";
 import { useTranslation } from "react-i18next";
 
+import { ExchangeImage } from "../ExchangeImage";
+
 type ExchangeCardProps = {
+	actions?: DropdownOption[];
 	exchange: any;
-	isSelected?: boolean;
 	onClick: any;
-};
-
-export const AddExchangeCard = ({ onAddExchange }: { onAddExchange: any }) => {
-	const { t } = useTranslation();
-
-	return (
-		<Card className="flex col-span-2" onClick={onAddExchange}>
-			<div className="flex items-center p-2 font-semibold text-theme-primary-600">
-				<Icon
-					name="Plus"
-					className="p-4 mr-4 rounded-lg bg-theme-primary-100 dark:border-theme-secondary-800"
-				/>
-
-				{t("EXCHANGE.ADD_EXCHANGE")}
-			</div>
-		</Card>
-	);
+	onSelect: any;
 };
 
 export const BlankExchangeCard = () => {
 	const { t } = useTranslation();
 
 	return (
 		<Card>
-			<div className="flex items-center p-2 font-semibold text-theme-primary-100 dark:text-theme-secondary-800">
-				<div className="mr-4 w-12 h-12 rounded-lg border-2 border-theme-primary-100 dark:border-theme-secondary-800" />
-
-				{t("EXCHANGE.EXCHANGE_NAME")}
+			<div className="flex items-center space-x-4">
+				<div className="h-25 w-25 rounded-lg border-2 border-theme-primary-100 dark:border-theme-secondary-800" />
```

Comment:
could probably refactor these placeholder images to a `placeholder` component so they can be reused 🤔